config/examples/sandbox_proxy.yml

```diff
@@ -28,25 +22,12 @@ development:
     - echo-api.3scale.net
 
 test:
+  <<: *default
   ignore_test_failures: []
-  test_api_hosts:
```

Comment:
This is same as default.
In any case, this seems to be used just to distinguish "test" calls from "real API", for analytics purposes